components/fd/dmesg_matcher.go

```diff
@@ -11,7 +11,9 @@ const (
 	//
 	// ref.
 	// https://docs.kernel.org/admin-guide/sysctl/fs.html#file-max-file-nr
-	RegexVFSFileMaxLimitReached = `VFS: file-max limit \d+ reached`
+	EventVFSFileMaxLimitReached   = "error_vfs_file_max_limit_reached"
```

Comment:
shall we remove the error prefix? We have the Event.Type.

Reply:
@cardyok Please confirm if it's ok

Reply:
yea its is ok